setup/README.md

```diff
@@ -101,13 +108,18 @@ db.createUser(
 
 
 ### 8. Setup meteor service variables in ~/service/myapp-meteor/run using above ports and mongo credentials
+If you use a custom domain (step 11) you can also use your own domain as `DDP_DEFAULT_CONNECTION_URL` and `ROOT_URL`. Make sure you have a valid HTTPS certificate then. 
+
+It's explained how to get one here for your own domain: https://wiki.uberspace.de/webserver:https .
+
+If you don't want to use HTTPS, just write `https://...`for both URLs. 
```

Comment:
Shouldn't it say "..., just write `http://...`for both URLs" without "s"?

Reply:
Yes. Should be corrected in my second commit (hopefully).